src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.IndexerConstants.DEVICE_ID_BELT;
+
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.robot.Constants;
+
+public class IndexerSubsystem implements Subsystem {
```

Comment:
Our Best Practices say we will use the Google Java Style Guide, and it says public classes need a JavaDoc comment.
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required